CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/visual/renderables/HitTester.java

```diff
@@ -64,7 +66,7 @@
     // The buffer to read from. This is hardcoded as their seems to be no real reason to change it,
     // but perhaps it should be looked up from the corersponding GraphRenderable. At the moment it simply
     // matches the buffer name used inside the if(doHitTesting) {} block of GraphRenderable's display method.
-    private final int hitTestBufferName = GL3.GL_COLOR_ATTACHMENT0;
```

Comment:
More curious as to what is happening here, was it picked up in Sonar?
Is it the below rule?
"static" base class members should not be accessed via derived types

Reply:
@aldebaran30701 Yes that is the rule

Reply:
yep